classes/controller/menu/class-postmeta-check-controller.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Post meta duplicate records check class
+ *
+ * @package P4BKS\Controllers\Menu
+ */
+
+namespace P4GBKS\Controllers\Menu;
+
+use P4GBKS\Command\Remove_Duplicate_Postmeta;
+
+if ( ! class_exists( 'Postmeta_Check_Controller' ) ) {
```

Comment:
You can remove this `class_exists` check, it doesn't load this file more than once.